server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxSetUpdatePerformer.scala

```diff
@@ -75,6 +75,9 @@ object MailboxSetUpdatePerformer {
       case e: InvalidPatchException =>
         LOGGER.info("Unsupported patch in Mailbox/set update: {}", e.getMessage)
         SetError.invalidPatch(SetErrorDescription(s"${e.cause}"))
+      case e if e.isInstanceOf[IllegalArgumentException] && e.getMessage == "Domain parts ASCII chars must be a-z A-Z 0-9 - or _" =>
+        LOGGER.info("Unsupported patch in Mailbox/set update: {}", e.getMessage)
```

Comment:
```suggestion
        LOGGER.info("Unsupported patch in Mailbox/set update", e)
```

Otherwise stacktrace is lost...